docs/for-ops/how-to/manage-age.md

```diff
@@ -0,0 +1,162 @@
+---
+slug: manage-age
+title: Manage Age for Development, Security, and Recovery
+sidebar_label: Manage Age
+---
+
+:::info
+This documentation was written for SOPS that is using Age Encryption. Please check the SOPS page for other [KMS](/docs/get-started/installation/sops#use-sops-with-an-external-key-management-service-kms) providers.
+:::
+
+## Prerequisites
+
+- Version >= `v3.1`
+
+## Local development setup
+
+### Steps
+
+1. Install [Age](https://github.com/FiloSottile/age/):
+
+```bash
+brew install age
+```
+
+2. Verify Age Versions:
+
+```bash
+age --version
+age-keygen --version
+```
+
+3. Install [Dependencies](https://github.com/linode/apl-core/blob/main/bin/install-deps.sh):
+
+```bash
+helm plugin install https://github.com/databus23/helm-diff.git
+helm plugin install https://github.com/jkroepke/helm-secrets.git --version v3.15.0
+```
+
+4. Create a values folder:
+
+```bash
+mkdir -p values-folder ~/workspace
+```
+
+5. Clone the values repo from Gitea:
+
+:::info
+Don't forget to set GIT_SSL_NO_VERIFY!
+```bash
+export GIT_SSL_NO_VERIFY=true
+```
+:::
+
+```bash
+git clone gitea.<cluster.domainSuffix>/otomi/values.git ~/workspace/values-folder
+```
+
+6. Copy SOPS_AGE_KEY secret:
+
+- Copy the `SOPS_AGE_KEY` secret, which is the private key for Age, from the cluster. It is located in the `otomi-pipelines` namespace in the `otomi-sops-secrets`, or in the `otomi` namespace in the `otomi-api` secret.
+
+- Create a `.secrets` file in the values-folder at the root level and insert the `SOPS_AGE_KEY` into it.
+
+:::info
+Don't forget to use = instead of :
+```bash
+SOPS_AGE_KEY=AGE-SECRET-KEY-1XXX
+```
+:::
+
+7. Export ENV_DIR:
+
+Export `ENV_DIR` in the core repository to use `otomi-cli`.
+
+```bash
+export ENV_DIR=~/workspace/values-folder
+```
+
+8. Use `otomi-cli`:
+
+The local development environment is now ready to use [otomi-cli](/docs/for-ops/cli/installation) commands.
+
+## Age keys rotation
+
+:::info
+Please follow the local development guidelines and be prepared to use `otomi-cli`.
+:::
+
+### Steps
+
+1. Decrypt values with old key and credentials:
+
+```bash
+otomi decrypt
```

Comment:
In order to have a more reliable experience with otomi-cli we should suggest the user to use the apl-core image. 
```suggestion
docker run -it -v $ENV_DIR:/home/app/stack/env linode/apl-core binzx/otomi decrypt
```